schema/ext.keboola.stream.sourceUndelete.json

```diff
@@ -34,10 +34,41 @@
                 "streamId",
                 "sourceName"
             ]
+        },
+        "results": {
```

Comment:
Why is it duplicated here the other way?

Reply:
I'm assuming it's necessary in order to avoid a time period where the events don't work correctly for billing? See https://github.com/keboola/event-schema/pull/22#discussion_r1824555875

Reply:
(I duplicated it for the new events as well so that I don't need to treat them differently in the code.)

Reply:
> (I duplicated it for the new events as well so that I don't need to treat them differently in the code.)

Aha! That was the missing piece ! :) Thanks